doc/whats_new/v1.6.rst

```diff
@@ -35,6 +35,7 @@ See :ref:`array_api` for more details.
 - :func:`sklearn.metrics.mean_tweedie_deviance` now supports Array API compatible
   inputs.
   :pr:`28106` by :user:`Thomas Li <lithomas1>`
+- :func:`sklearn.metrics.mean_absolute_error` :pr:`27736` by :user:`Edoardo Abati <EdAbati>`.
```

Comment:
```suggestion
  :pr:`28106` by :user:`Thomas Li <lithomas1>`

- :func:`sklearn.metrics.mean_absolute_error` :pr:`27736` by :user:`Edoardo Abati <EdAbati>`.
```

Reply:
In version 1.4 we didn't have the new line between these lines: https://github.com/scikit-learn/scikit-learn/blob/main/doc/whats_new/v1.4.rst?plain=1#L393

should we update anyway?

Reply:
I think we can keep it then.